meta/autoid/autoid.go

```diff
@@ -76,6 +78,18 @@ func (alloc *allocator) End() int64 {
 	return alloc.end
 }
 
+// NextGlobalAutoID implements autoid.Allocator NextGlobalAutoID interface.
+func (alloc *allocator) NextGlobalAutoID(tableID int64) (int64, error) {
+	var autoID int64
+	err := kv.RunInNewTxn(alloc.store, true, func(txn kv.Transaction) error {
+		var err1 error
+		m := meta.NewMeta(txn)
+		autoID, err1 = m.GetAutoTableID(alloc.dbID, tableID)
```

Comment:
`autoID, err1 := m.GetAutoTableID(alloc.dbID, tableID)`, remove line 85

Reply:
I need to return autoID at the end of `NextGlobalAutoID `.